__tests__/app.test.js

```diff
@@ -8,7 +8,18 @@ afterAll(() => db.end());
 
 beforeEach(() => seed(testData));
 
-describe("/api/topics", () => {
+describe("ALL /*", () => {
+  test("Status: 404 with an error message for a none-existing endpoint", () => {
+    return request(app)
+      .get("/api/invalid-endpoint")
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe("Sorry, request invalid..");
```

Comment:
On a personal note, I think it's THEM who should be sorry, not you - but the code is good.